romeo_sensors_py/setup.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python
```

Comment:
If you have Python, you need to update your CMakeLists.txt:
http://docs.ros.org/jade/api/catkin/html/user_guide/setup_dot_py.html

You should not install scripts through setup.py a mentioned on that page too.